.github/workflows/tests.yml

```diff
@@ -30,6 +30,9 @@ jobs:
   build:
     runs-on: ubuntu-latest
 
+    env:
+      CI: true
```

Comment:
this should be set by github actions already automatically

Reply:
ok, will remove it then. I had just moved it from the steps level to the job level.